kuksa-client/docs/examples/async-grpc.md

```diff
@@ -34,6 +36,21 @@ async def main():
 asyncio.run(main())
 ```
 
+Besides this there is a solution where you are not using the client as context-manager
+```python
+import asyncio
+
+from kuksa_client.grpc.aio import VSSClient
+
+async def main():
+    client = VSSClient('127.0.0.1', 55555, ensure_startup_connection=True)
```

Comment:
why do wen need ensure_startup_connection=True?

Reply:
We do not need it there just wanted to mention it somewhere. The default of this is True.